upup/pkg/fi/cloudup/openstack/cloud.go

```diff
@@ -388,8 +380,30 @@ func NewOpenstackCloud(tags map[string]string, spec *kops.ClusterSpec) (Openstac
 			}
 			spec.CloudConfig.Openstack.Loadbalancer.FloatingNetworkID = fi.String(lbNet[0].ID)
 		}
+		if spec.CloudConfig.Openstack.Loadbalancer.UseOctavia != nil {
+			octavia = fi.BoolValue(spec.CloudConfig.Openstack.Loadbalancer.UseOctavia)
+		}
 	}
 
+	var lbClient *gophercloud.ServiceClient
+	if octavia {
+		glog.V(2).Infof("Openstack using Octavia loadbalancer")
+		lbClient, err = os.NewLoadBalancerV2(provider, gophercloud.EndpointOpts{
+			Region: region,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("error building lb client: %v", err)
+		}
+	} else {
+		glog.V(2).Infof("Openstack using old lbaasv2 loadbalancer")
```

Comment:
nit: I think mentioning the other loadbalancer as "old" is a little harsh. Its either an alternative or deprecated.